tests/keras/layers/test_advanced_activations.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+from numpy.testing import assert_allclose
+import numpy as np
+from keras import backend as K
+
+np.random.seed(1337)
+
+def get_standard_values():
+    '''
+    These are just a set of floats used for testing the activation
+    functions, and are useful in multiple tests.
+
+    The values should all be non-negative becasue they and their negative
+    are used to test the ReLU derivates in this file.
+    '''
+    return np.array([[0, 0.1, 0.5, 0.9, 1.0, 10, 1e3, 0.0001]], dtype=K.floatx())
+
+class TestAdvancedActivations(unittest.TestCase):
```

Comment:
whats the value of using unittest here? As I know keras uses pytest. Mixing pytest and unittest complicates subsequent support: the same feature can implemented with either framework, etc. (plus you have to add self to every test without any value)